sdks/python/apache_beam/ml/inference/sklearn_loader_test.py

```diff
@@ -151,6 +151,12 @@ def test_bad_file_raises(self):
             SklearnModelLoader(model_uri='/var/bad_file_name'))
         pipeline.run()
 
+  def test_bad_input_type_raises(self):
+    with tempfile.NamedTemporaryFile() as file:
+      with self.assertRaises(TypeError):
```

Comment:
just to be super clear that it's the custom error that you wrote, it might be better to do `self.assertRaisesRegex(TypeError, 'Unsupported serialization type.')`